dataland-document-manager/src/main/resources/application.properties

```diff
@@ -30,8 +30,8 @@ dataland.authorization.apikey.enable=true
 dataland.apikeymanager.base-url=http://api-key-manager:8080/api-keys
 dataland.internalstorage.base-url=http://internal-storage:8080/internal-storage
 
-spring.servlet.multipart.max-file-size=${DOCUMENT_UPLOAD_MAX_FILE_SIZE_IN_MEGABYTE:100}MB
-spring.servlet.multipart.max-request-size=${DOCUMENT_UPLOAD_MAX_FILE_SIZE_IN_MEGABYTE:100}MB
+spring.servlet.multipart.max-file-size=${DOCUMENT_UPLOAD_MAX_FILE_SIZE_IN_MEGABYTES:100}MB
+spring.servlet.multipart.max-request-size=${DOCUMENT_UPLOAD_MAX_FILE_SIZE_IN_MEGABYTES:100}MB
```

Comment:
why 100MB instead of 150?

Reply:
I removed that which caused some problems with environments that have not set this variable, i.e. the no-db profile and unit tests that used the default spring profile. Thus I added the constants to the no-db profile such that ./generateOpenApiDocs works and also let the document manager unit tests run with this profile.